editor/animation_track_editor.cpp

```diff
@@ -1997,10 +2005,19 @@ AnimationTimelineEdit::AnimationTimelineEdit() {
 	play_position->set_anchors_and_offsets_preset(PRESET_FULL_RECT);
 	play_position->connect(SceneStringName(draw), callable_mp(this, &AnimationTimelineEdit::_play_position_draw));
 
+	add_track_hb = memnew(HBoxContainer);
+	add_child(add_track_hb);
+
 	add_track = memnew(MenuButton);
 	add_track->set_position(Vector2(0, 0));
```

Comment:
Consider adding `add_track->set_tooltip_text(TTR("Select a new track by type to add to this animation."));` since while its functionality seems mostly self-evident, we no longer explicitly label what this button does, and this is standard practice for most UI related things.

Reply:
Good point. Done! :)